spec/repositories/forms_repository_spec.rb

```diff
@@ -21,6 +21,13 @@
       expect(form[:submission_email]).to eq("submission_email")
       expect(form[:org]).to eq("org")
     end
+    it "gets a form by org" do
+      subject.create("name", "submission_email", "org")
```

Comment:
Might be worth making a couple to make sure it returns two?

Reply:
Very smart, added that